### PR TITLE
convert any uppercase to lower case

### PR DESCRIPTION
**Pull request by karim20230**
Fix terraform palyground WORKSPACE name uppercase characters to lower…

### CODE REVIEW

File: terraform/playground/terraform-playground.sh [hunk 1]
```diff
@@ -58,7 +58,7 @@ generate_azure_private_workspace_name () {
     # becomes 'foobar'.
     # Below command errors out with the azure error message if the azure user
     # is not signed-in.
-    WORKSPACE=$(az ad signed-in-user show | jq -cr .userPrincipalName | cut -d'@' -f1 | sed 's/[^a-zA-Z0-9]//g' | cut -c 1-20)
+   WORKSPACE=$(az ad signed-in-user show | jq -cr .userPrincipalName | cut -d'@' -f1 | sed 's/[^a-zA-Z0-9]//g' | cut -c 1-20 | tr '[:upper:]' '[:lower:]')
```
Comment (henrirosten) [inline terraform/playground/terraform-playground.sh:61]: Please fix the indentation (add one space character before `WORKSPACE=...`)

Comment (henrirosten) [reply terraform/playground/terraform-playground.sh:61]: Also, for the sake of symmetry, please consider adding the following on line 150:
`exit_unless_command_exists tr`

And the following comment on line 57:
`# - tr converts the string to lower case`

**Review by henrirosten — CHANGES_REQUESTED** [review]
Please fix the indentation (add one space character before WORKSPACE=)

File: terraform/playground/terraform-playground.sh (hunk 1 above)
Comment (henrirosten) [reply terraform/playground/terraform-playground.sh:61]: Also, not enforced currently, but you should try to follow the commit message guidelines at: https://github.com/tiiuae/ghaf/blob/main/CONTRIBUTING.md#commit-message-guidelines.
In this case, while making the above suggested changes, you could also re-formulate the commit subject-line.

**Comment by karim20230** [comment]
thanks, addressed ur comments @henrirosten

**Review by mnokka — APPROVED** [review]
We need this logical change for Azure operations

**Comment by henrirosten** [comment]
@karim20230 : can you still squash the commits?

**Review by henrirosten — APPROVED** [review]

**Comment by mnokka-unikie** [comment]
Verified that this works totally, ok  it does lowercasing ok when setting up our Azure playground

**Pull request merged**